docs/fundamentals/code-analysis/quality-rules/performance-warnings.md

```diff
@@ -49,3 +49,4 @@ Performance rules support high-performance libraries and applications.
 | [CA1836: Prefer `IsEmpty` over `Count` when available](ca1836.md) | Prefer `IsEmpty` property that is more efficient than `Count`, `Length`, <xref:System.Linq.Enumerable.Count%60%601%28System.Collections.Generic.IEnumerable%7B%60%600%7D%29> or <xref:System.Linq.Enumerable.LongCount%60%601%28System.Collections.Generic.IEnumerable%7B%60%600%7D%29> to determine whether the object contains or not any items. |
 | [CA1837: Use `Environment.ProcessId` instead of `Process.GetCurrentProcess().Id`](ca1837.md) | `Environment.ProcessId` is simpler and faster than `Process.GetCurrentProcess().Id`. |
 | [CA1838: Avoid `StringBuilder` parameters for P/Invokes](ca1838.md) | Marshaling of `StringBuilder` always creates a native buffer copy, resulting in multiple allocations for one marshaling operation. |
+| [CA1839: Use `string.Contains(char)` instead of `string.Contains(string)` with single characters](ca1839.md) | Consider using `string.Contains(char)` instead of `string.Contains(string)` when searching for a single character. |
```

Comment:
Can you also add it to the overall index [here](https://github.com/dotnet/docs/blob/master/docs/fundamentals/code-analysis/quality-rules/index.md), and to the table of contents [here](https://github.com/dotnet/docs/blob/master/docs/fundamentals/toc.yml).